src/internal/progress/progress.go

```diff
@@ -86,6 +86,18 @@ func Open(path string) (*File, error) {
 	return newFile(file, path, fi.Size()), nil
 }
 
+func OpenAppend(path string, appendSize int64) (*File, error) {
+	file, err := os.OpenFile(path, os.O_APPEND|os.O_RDWR, 0666)
```

Comment:
Is read permission required for the progress bar?

Reply:
it actually isn't necessary -  changed it to write only